### ISSUE TITLE
Add support for Bonne projection

### ISSUE THREAD

**Issue by mclaeysb** [proj4js/proj4js#472]
The [Bonne projection](https://en.wikipedia.org/wiki/Bonne_projection) is [defined](https://proj.org/en/9.4/operations/projections/bonne.html?highlight=bonne) and [encoded](https://github.com/OSGeo/PROJ/blob/ea00f187f7572b3b745ed41e0285d6223a2ec488/src/projections/bonne.cpp) in PROJ but not yet in Proj4.js. 

At [Allmaps](allmaps.org) we'd like to use this projection a.o. in a pipeline to process a [historical map series from the Netherlands](https://observablehq.com/@allmaps/bonnebladen).

As a test-case, in PROJ the result of 

```sh
echo 4.9 52.366667 | gdaltransform -t_srs '+proj=bonne +lat_1=10 +lon_0=10' -s_srs 'EPSG:4326'
```

is `-347381.937958562 4700204.94589969 0`

**Comment by ahocevar** [comment]
@mclaeysb Pull request #473 implements the Bonne projection. Are you able to test with the pull request branch directly, or do you need a release?

**Comment by mclaeysb** [comment]
Thank you for this very quick action @ahocevar ! Our further testing would be in a observable notebook, so a release would be much more practical.

**Comment by ahocevar** [comment]
@mclaeysb Published v2.12.0.
